docs/Config.md

```diff
@@ -113,6 +113,7 @@ keybinding:
     quit: 'q'
     quit-alt1: '<c-c>' # alternative/alias of quit
     return: '<esc>' # return to previous menu, will quit if there's nowhere to return
+    return-alt1: ';' # Can be set as a single character key to return to the previous menu level
```

Comment:
Why is the 'single-character' part important?

Reply:
Because I'm lazy and want to press as few buttons as possible.

Reply:
Maybe it's a problem with my presentation, because `return` can't be set to a single character, otherwise it won't be returned in the input box.

Reply:
Maybe we should make that comment more explicit e.g.:
```
# When set to a printable character, this will work for returning from non-prompt panels
```